libs/ui-lib/lib/ocm/components/clusterWizard/OperatorsSelect.tsx

```diff
@@ -24,10 +24,12 @@ const OperatorsSelect = ({
   cluster,
```

Comment:
Lets not just add to existing code, but rethink how this should now be done in a logical order
1. filter
2. group
3. sort within each group

Reply:
@rawagner let's talk about that when you can